saleor/graphql/core/types/sort_input.py

```diff
@@ -35,3 +35,13 @@ def __init_subclass_with_meta__(
                 description=f"Sort {type_name} by the selected field.",
             )
             cls._meta.fields.update({"field": field})
+
+
+class SortInputWitchChannelObjectType(SortInputObjectType):
```

Comment:
There is a typo in "Witch" - it should be "With". But I would suggest a different name: `ChannelSortInputObjectType`